src/controllers/controller.bar.js

```diff
@@ -227,7 +239,7 @@ module.exports = function(Chart) {
 			rectangle._model = {
 				datasetLabel: dataset.label,
 				label: chart.data.labels[index],
-				borderSkipped: custom.borderSkipped ? custom.borderSkipped : rectangleOptions.borderSkipped,
+				borderSkipped: custom.borderSkipped || custom.borderSkipped === null ? custom.borderSkipped : rectangleOptions.borderSkipped,
```

Comment:
I don't understand what you're checking for here. Won't `custom.borderSkipped || custom.borderSkipped === null` almost always be true? Are you trying to do `custom.borderSkipped !== undefined` or `custom.borderSkipped !== false`? We rarely would check for `custom.borderSkipped === null` and instead would usually do `helpers.isNullOrUndef(custom.borderSkipped)`

Reply:
Ah, I see. I think it's better not to treat `null` and `undefined` differently, so a better way to do it could be to have a default value of `left` or `bottom` here as appropriate (depending on whether it's a horizontal or vertical chart). Then in element.rectangle.js you can treat `null` and `undefined` both as skipping no border. @simonbrunel @etimberg what do you think?

Reply:
previously it was like 'none', so if value = none, we should draw all sides. But we discussed this somewhere in the top and @simonbrunel advised this:

I would not introduce the new borderSkipped: 'none' value, but instead support borderSkipped: false|null

So, we changed none to support null instead. For now rectangle data is passing from controller.bar.js to element.rectangle.js to initialize bar to be drawn. And borderskipped var is also passed from controller to rectangle script with some default value  - left or bottom in case horizontal/vertical bars, so its better i think to pass modifed value for borderskipped from controller (null) to tell no skipping for bar borders required. Its better to do from controller where we have already getting dataset values and can check if the bar values are for floating bar or not and put borderskipped to proper side or null. This is how it works now.

Reply:
done